website/content/docs/internals/telemetry.mdx

```diff
@@ -375,54 +376,72 @@ These metrics relate to the supported [storage backends][storage-backends].
+| `vault.raft.verify_leader`                                             | Number of times node checks whether it is still the leader or not.                                                                                                                                                | checks / interval                 | counter |
+| `vault.raft-storage.delete`                                            | Time to insert log entry to delete path.                                                                                                                                                                          | ms                                | timer   |
+| `vault.raft-storage.get`                                               | Time to retrieve value for path from FSM.                                                                                                                                                                         | ms                                | timer   |
+| `vault.raft-storage.put`                                               | Time to insert log entry to persist path.                                                                                                                                                                         | ms                                | timer   |
+| `vault.raft-storage.list`                                              | Time to list all entries under the prefix from the FSM.                                                                                                                                                           | ms                                | timer   |
+| `vault.raft-storage.transaction`                                       | Time to insert operations into a single log.                                                                                                                                                                      | ms                                | timer   |
+| `vault.raft-storage.entry_size`                                        | The total size of a Raft entry during log application in bytes.                                                                                                                                                   | bytes                             | summary  |
+| `vault.raft_storage.bolt.freelist.`<br/>`free_pages`                          | Number of free pages in the freelist.                                                                                                                                                                      | pages                             | gauge  |
```

Comment:
Ditto the bolt metrics.

Reply:
Aha! That's why they weren't published, gotcha, thanks!!